modules_core/org.openbravo.client.application/web/org.openbravo.client.application/js/process/ob-parameter-window-view.js

```diff
@@ -484,7 +484,7 @@ isc.OBParameterWindowView.addProperties({
         },
         function(rpcResponse, data, rpcRequest) {
           view.handleResponse(
-            !(data && data.refreshParent === false),
+            !(data && !data.refreshParent),
```

Comment:
The double negation is a bit confusing, subject to testing, isn't it equivalent to: "data && data.refreshParent !== true"?

Reply:
Yes, it's equivalent. I have just changed the logic because "undefined === false" is not the same that "false === false". I'll change it to data.refreshParent !== true. It should be the same